src/core_atmosphere/Registry.xml

```diff
@@ -1857,6 +1922,16 @@
                      description="closure option for shallow convection in Grell-Freitas convection scheme"
                      possible_values="8"/>
 
+                <nml_option name="config_mynn_cloudpdf" type="integer" default_value="-1" in_defaults="false"
+                     units="-"
+                     description="switch to different cloud PDFs to represent subgrid clouds"
+                     possible_values="0,1,2"/>
+
+                <nml_option name="config_mynn_edmf" type="integer" default_value="0" in_defaults="false"
+                     units="-"
+                     description="option to activate the PBL eddy diffusivity mass flux scheme"
+                     possible_values="0,1"/>
+
```

Comment:
These look like they belong to the set of changes to the MYNN scheme, and not Noah.